src/avram/save_operation.cr

```diff
@@ -44,6 +44,10 @@ abstract class Avram::SaveOperation(T)
   abstract def primary_key_name
   abstract def database
 
+  def self.param_key
+    T.name.underscore
+  end
```

Comment:
Now we use the model class. So `TaxId` model would have a `param_key` of `tax_id` no matter what the operation is named. I think this makes a lot more sense and keeps things consistent and easy to learn